services/src/main/java/org/keycloak/services/resources/KeycloakApplication.java

```diff
@@ -109,7 +108,7 @@ public KeycloakApplication() {
             classes.add(KcUnrecognizedPropertyExceptionHandler.class);
 
             singletons.add(new ObjectMapperResolver());
```

Comment:
Why this `ObjectMapperResolver` is represented as a singleton? Would probably be better to use the same approach as for the WelcomeResource, right? Add it here to classes, and in QuarkusKeycloakApplication remove it. The instantiation of this object is done here and for Quarkus not used at all. 

WDYT?

Reply:
I would prefer to avoid changing `KeycloakApplication` in this PR. It is still used by Undertow and in Quarkus we do it right.

Reply:
Yeah, I understand, but why are the other singletons changed here in KeycloakApplication? It's only a nitpick, but would be good to have it unified, and I'd like to find out if there's some special intention behind it :))

Reply:
I remember facing some errors when doing it. I need to change and check again.